dali/datest/datest.cpp

```diff
@@ -958,7 +958,7 @@ void TestRemoteFile(unsigned part,unsigned of)
 
 void QTest(bool testput)
 {
-    PrintLog("starting QTest %s",testput?"put":"get");
+    DBGLOG("starting QTest %s",testput?"put":"get");
```

Comment:
Not sure about any (most) of the changes to DBGLOG in this file.
We prob. want to see datest test tracing even in release, so wouldn't want these to go away if we start suppressing DBGLOG.

Bettter to change them all to either PROGLOG I think.